address;source_location;insn;indent
0x500f20;/usr/local/src/conda/python-3.9.19/Python/errors.c:299;PUSH	%R13;
0x500f22;:0;PUSH	%R12;
0x500f24;:0;PUSH	%RBP;
0x500f25;:0;PUSH	%RBX;
0x500f26;:0;SUB	$0x8,%RSP;
0x500f2a;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_pystate.h:55;MOV	0x247027(%RIP),%RAX;
0x500f31;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_pyerrors.h:14;MOV	0x58(%RAX),%RBP;
0x500f35;/usr/local/src/conda/python-3.9.19/Python/errors.c:261;TEST	%RBP,%RBP;
0x500f38;:0;JE	501000 <PyErr_ExceptionMatches+0xe0>;
0x500f3e;:0;MOV	%RDI,%RBX;
0x500f41;:0;TEST	%RDI,%RDI;
0x500f44;:0;JE	501000 <PyErr_ExceptionMatches+0xe0>;
0x500f4a;/usr/local/src/conda/python-3.9.19/Include/object.h:628;MOV	0x8(%RDI),%RDX;
0x500f4e;:0;MOV	0xa8(%RDX),%R8;
0x500f55;/usr/local/src/conda/python-3.9.19/Python/errors.c:265;TEST	$0x4000000,%R8D;
0x500f5c;:0;JNE	501020 <PyErr_ExceptionMatches+0x100>;
0x500f62;/usr/local/src/conda/python-3.9.19/Python/errors.c:279;MOV	0x8(%RBP),%RCX;
0x500f66;/usr/local/src/conda/python-3.9.19/Include/object.h:628;MOV	0xa8(%RCX),%RSI;
0x500f6d;/usr/local/src/conda/python-3.9.19/Python/errors.c:279;TEST	$0x40000000,%ESI;
0x500f73;:0;JNE	501065 <PyErr_ExceptionMatches+0x145>;
0x500f79;/usr/local/src/conda/python-3.9.19/Python/errors.c:282;TEST	$-0x80000000,%ESI;
0x500f7f;:0;JE	431dd1 <PyErr_ExceptionMatches.cold+0x3>;
0x500f85;/usr/local/src/conda/python-3.9.19/Python/errors.c:282;TESTB	$0x40,0xab(%RBP);
0x500f8c;:0;JE	431dd1 <PyErr_ExceptionMatches.cold+0x3>;
0x500f92;/usr/local/src/conda/python-3.9.19/Python/errors.c:282;TEST	$-0x80000000,%R8D;
0x500f99;:0;JE	431dd1 <PyErr_ExceptionMatches.cold+0x3>;
0x500f9f;/usr/local/src/conda/python-3.9.19/Python/errors.c:282;TESTB	$0x40,0xab(%RBX);
0x500fa6;:0;JE	431dd1 <PyErr_ExceptionMatches.cold+0x3>;
0x500fac;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1425;MOV	0x158(%RBP),%R9;
0x500fb3;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1426;TEST	%R9,%R9;
0x500fb6;:0;JE	501046 <PyErr_ExceptionMatches+0x126>;
0x500fbc;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1431;MOV	0x10(%R9),%R10;
0x500fc0;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1432;TEST	%R10,%R10;
0x500fc3;:0;JLE	501000 <PyErr_ExceptionMatches+0xe0>;
0x500fc5;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1433;CMP	0x18(%R9),%RBX;
0x500fc9;:0;JE	501010 <PyErr_ExceptionMatches+0xf0>;
0x500fcb;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1432;CMP	$0x1,%R10;
0x500fcf;:0;JE	501000 <PyErr_ExceptionMatches+0xe0>;
0x500fd1;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1433;CMP	0x20(%R9),%RBX;
0x500fd5;:0;JE	501010 <PyErr_ExceptionMatches+0xf0>;
0x500fd7;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1432;CMP	$0x2,%R10;
0x500fdb;:0;JE	501000 <PyErr_ExceptionMatches+0xe0>;
0x500fdd;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1433;CMP	0x28(%R9),%RBX;
0x500fe1;:0;JE	501010 <PyErr_ExceptionMatches+0xf0>;
0x500fe3;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1432;CMP	$0x3,%R10;
0x500fe7;:0;JE	501000 <PyErr_ExceptionMatches+0xe0>;
0x500fe9;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1432;MOV	$0x3,%R11D;
0x500fef;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1433;CMP	0x18(%R9,%R11,8),%RBX;(1073) 
0x500ff4;:0;JE	501010 <PyErr_ExceptionMatches+0xf0>;(1073) 
0x500ff6;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1432;INC	%R11;(1073) 
0x500ff9;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1432;CMP	%R11,%R10;(1073) 
0x500ffc;:0;JNE	500fef <PyErr_ExceptionMatches+0xcf>;(1073) 
0x500ffe;:0;XCHG	%AX,%AX;
0x501000;/usr/local/src/conda/python-3.9.19/Python/errors.c:301;XOR	%EAX,%EAX;
0x501002;/usr/local/src/conda/python-3.9.19/Python/errors.c:302;ADD	$0x8,%RSP;
0x501006;:0;POP	%RBX;
0x501007;:0;POP	%RBP;
0x501008;:0;POP	%R12;
0x50100a;:0;POP	%R13;
0x50100c;:0;RET;
0x50100d;:0;NOPL	(%RAX);
0x501010;/usr/local/src/conda/python-3.9.19/Python/errors.c:302;ADD	$0x8,%RSP;
0x501014;/usr/local/src/conda/python-3.9.19/Python/errors.c:301;MOV	$0x1,%EAX;
0x501019;/usr/local/src/conda/python-3.9.19/Python/errors.c:302;POP	%RBX;
0x50101a;/usr/local/src/conda/python-3.9.19/Python/errors.c:302;POP	%RBP;
0x50101b;:0;POP	%R12;
0x50101d;:0;POP	%R13;
0x50101f;:0;RET;
0x501020;/usr/local/src/conda/python-3.9.19/Python/errors.c:267;CALL	4ef400 <PyTuple_Size>;
0x501025;/usr/local/src/conda/python-3.9.19/Python/errors.c:268;XOR	%R12D,%R12D;
0x501028;/usr/local/src/conda/python-3.9.19/Python/errors.c:267;MOV	%RAX,%R13;
0x50102b;/usr/local/src/conda/python-3.9.19/Python/errors.c:268;CMP	%R12,%R13;(1071) 
0x50102e;:0;JLE	501000 <PyErr_ExceptionMatches+0xe0>;(1071) 
0x501030;/usr/local/src/conda/python-3.9.19/Python/errors.c:270;MOV	0x18(%RBX,%R12,8),%RSI;(1071) 
0x501035;:0;MOV	%RBP,%RDI;(1071) 
0x501038;:0;CALL	501090 <PyErr_GivenExceptionMatches>;(1071) 
0x50103d;/usr/local/src/conda/python-3.9.19/Python/errors.c:270;TEST	%EAX,%EAX;(1071) 
0x50103f;:0;JNE	501010 <PyErr_ExceptionMatches+0xf0>;(1071) 
0x501041;/usr/local/src/conda/python-3.9.19/Python/errors.c:268;INC	%R12;(1071) 
0x501044;/usr/local/src/conda/python-3.9.19/Python/errors.c:268;JMP	50102b <PyErr_ExceptionMatches+0x10b>;(1071) 
0x501046;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1412;CMP	%RBP,%RBX;(1072) 
0x501049;:0;JE	501010 <PyErr_ExceptionMatches+0xf0>;(1072) 
0x50104b;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1414;MOV	0x100(%RBP),%RBP;(1072) 
0x501052;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1415;TEST	%RBP,%RBP;(1072) 
0x501055;:0;JNE	501046 <PyErr_ExceptionMatches+0x126>;(1072) 
0x501057;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1417;XOR	%EAX,%EAX;
0x501059;/usr/local/src/conda/python-3.9.19/Objects/typeobject.c:1417;CMP	$0x738c80,%RBX;
0x501060;:0;SETE	%AL;
0x501063;:0;JMP	501002 <PyErr_ExceptionMatches+0xe2>;
0x501065;/usr/local/src/conda/python-3.9.19/Include/object.h:628;MOV	0x8(%RCX),%RDI;
0x501069;/usr/local/src/conda/python-3.9.19/Python/errors.c:282;TESTB	$-0x80,0xab(%RDI);
0x501070;:0;JE	431dce <PyErr_ExceptionMatches.cold>;
0x501076;/usr/local/src/conda/python-3.9.19/Include/object.h:630;MOV	%RCX,%RBP;
0x501079;:0;JMP	500f92 <PyErr_ExceptionMatches+0x72>;
0x50107e;/usr/local/src/conda/python-3.9.19/Include/object.h:630;XCHG	%AX,%AX;
